### PR TITLE
Add media proposal

### PR DESCRIPTION
**Pull request by Citrullin**
We had some discussions in #94. There is a lack of media coverage on the Web of Things.
I wrote something down and also added some relevant media. Please take it with a grain of salt. This is not a complete media strategy with all details. 
See this more as a starting point. I don't have the time to get more into details.

### CODE REVIEW

**Comment by egekorkan** [comment]
Thanks a lot for the proposal! :) 

We haven't proactively tried reaching out to media outlets but it is very important I think. (There was an article by Heise.de as far as I know).

Regarding a better website, it is sadly in a deadlock situation for _some time_.

File: proposals/media-strategy/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+These frameworks have in common to have an appealing website. But this is not only common in the Javascript ecosystem. 
+It is a trend to have well designed websites, across frameworks. Even recent security bugs get [well designed Logos and a website.](https://spectreattack.com/)
+
+# Website
+
+The user attention is valuable and should be treated as such. In a network where an information is just one click away, users want to get the information they need as fast as possible. 
+A website should take this into account. Just the VueJS and React.js website as an example.
+Within the users viewport all necessary information are displayed.
+The user gets the following questions answered within seconds: "What do I look at?", "What is this framework/library about?" "Why should I use it?"
+This is necessary to keep the users attention and get them excited. Before going into the technical details, the user has to understand why this site is relevant for them. In case of the W3C WoT standard, this means: Don't focus on the standard itself, focus on what the user can do with the standard.
```
Comment (relu91) [inline proposals/media-strategy/README.md:15]: +1 this is a good guideline for the new website. We should take it in mind.

**Review by relu91 — APPROVED** [review]
It is a good start, plus it contains some nice guidelines for the new website. Let's start from here and improve!

**Comment by relu91** [comment]
> Regarding a better website, it is sadly in a deadlock situation for some time.

Yet things are starting to move. :) we for sure need to aim for a faster website development cycle. Probably a draft site rendered for each PR would help a lot to review them and approve.

**Comment by k-toumura** [comment]
> Probably a draft site rendered for each PR would help a lot to review them and approve.

FYI, [netlify](https://www.netlify.com/) provides 'Deploy Previews' function: [Introducing Deploy Previews in Netlify](https://www.netlify.com/blog/2016/07/20/introducing-deploy-previews-in-netlify/)

**Comment by Citrullin** [comment]
There is a [video ](https://www.youtube.com/watch?v=2_Si-iD8ZRQ) on the W3C Youtube channel which I really enjoyed. I can see having something similar on this Web of Things website. In order to explain what the standard solves etc.

**Comment by egekorkan** [comment]
> There is a video on the W3C Youtube channel I really enjoyed. I can see having something similar on this Web of Things website. In order to explain what the standard solves etc.

This is actually in works and almost finalized ;)

**Comment by Citrullin** [comment]
Google is killing [Android Things.](https://www.androidauthority.com/google-android-things-shutdown-1186745/) 
Could be interesting to address these kind of topics. Especially in combination with right to repair. This is a story you can sell. 
Being environmental responsible and supporting devices for a longer life-time etc.

**Comment by sebastiankb** [comment]
From today's marketing call we decided to merge this PR. @Citrullin thanks for the input

**Pull request merged**